src/core/reducer.js

```diff
@@ -322,6 +322,10 @@ export function CreateGameReducer({ game, multiplayer }) {
       }
 
       case Actions.UNDO: {
+        if (multiplayer) {
```

Comment:
I think we can allow these two actions, as long as we strip `_undo` and `_redo` like you are in `master.js`.  These fields are only needed by the current player (and do not need to be sent over the network).